.github/workflows/release.yml

```diff
@@ -15,22 +15,21 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
 
-      - name: Setup Node.js 16.x
+      - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16.x
+          node-version: 18
 
```

Comment:
Can we add the `Set up JDK` here as well, so it is consistent between both?

Reply:
Just added. Does that look right?